spec/index.md

```diff
@@ -3702,6 +3732,72 @@ The functions `query_as_update` and `system_task_as_update` turns a query functi
 
 Note that by construction, a query function will either trap or return with a response; it will never send calls, and it will never change the state of the canister.
 
+#### Spontaneous request rejection {#request-rejection}
+
+The system can reject a request at any point in time, e.g., because it is overloaded. The following specification is an overapproximation of the rejection behavior. In particular, the system is guaranteed not to emit `DESTINATION_INVALID` in some cases, e.g., if the callee exists, and if the caller successfully calls the caller once, subsequent calls will not fail with `DESTINATION_INVALID`.
```

Comment:
```suggestion
The system can reject a request at any point in time, e.g., because it is overloaded.
```

Reply:
I'd drop this as the spec generally overapproximates the actual behavior and the example is quite hard to follow.

Reply:
> if the caller successfully calls the caller once, subsequent calls will not fail with `DESTINATION_INVALID`.

this might not be true if the callee canister is deleted